### ISSUE TITLE
Yadda overwrites existing non-Yadda test

### ISSUE THREAD

**Issue by lolmaus** [albertjan/ember-cli-yadda#78]
Hi!

In my app I have a traditional `tests/acceptance/foo-test.js` and a new `tests/acceptance/foo.feature`.

Turns out, Yadda compiles the feature into a temporary file and puts it into the tests folder for QUnit to run.

The problem is that it overwrites the existing `tests/acceptance/foo-test.js`! The change persists since it's already under version control Tests still pass, so the autogenerated file can be checked into git by accident.

A simple solution could be to prefix autogenerated files with something a user would never write by hand, e. g. `__yadda__foo-test.js` instead of just `foo-test.js`.

**Comment by simonihmig** [comment]
I see two basically separate issues mentioned here:
* possibility of a name clash, which indeed can happen, and I don't see any reason not to implement some namespacing for the file like you suggested
* if I understand you correctly, the generated `tests/acceptance/foo-test.js` is written not into some temporary broccoli folder in `./tmp` (as it should be), but into your main project folder itself. Is that what you are saying? I can hardly imagine how this would be possible... 🤨

**Comment by lolmaus** [comment]
Yes and yes. My teammate @vvainio saw it as well on his machine.
